elizhahn/karaoke-stretch
# Styling/detailing

## What does this PR do (summary of changes)?

- Applies animation to the landing page and representative link on the Navigation component
- Replaces old image assets and adds icons that better align with the branding of the application
- Applies hover transitions and shadow on NavLinks
- Refactors some common styles into extends
- Media query is applied to SongCard component to achieve responsivity down to 360px width
- Moves App component into its own component directory and fixed its import paths
- Removes repetitive scss imports in component JS files

## How should it be tested?
 
- User can observe more visual cues that imply interactivity on links
- User can easily observe all relevant elements and reach them on all screen widths down to 360px (including mobile)

## Future Iterations:

 - Final styling PR
 - Future iterations of the application might have the Navigation component moved to the bottom of the page
